src/resources/assets/js/annotations/components/settingsTabPlugin.vue

```diff
@@ -0,0 +1,269 @@
+            let projection = new Projection({
+                //needs to be same code as in annotationCanvas.vue in biigle/core
+                code: 'biigle-image',
+                units: 'pixels',
+            });
+
+            let sourceLayer = new ZoomifySource({
+                    url: this.overlayUrlTemplate.replaceAll(':id', overlay.id),
+                    size: [overlay.attrs.width, overlay.attrs.height],
+                    crossOrigin: 'anonymous',
+                    zDirection: -1, // Ensure we get a tile with the screen resolution or higher
+                    projection: projection
+            });
+
+            // let extentPixel = [
+            // $top_left = [0, 0];
+            // $bottom_left = [0, $height];
+            // $top_right = [$width, 0];
+            // $bottom_right = [$width, $height];
+            // ];
+
+            // let extentEPSG4326 = [
+            //     overlay.attrs.top_left_lng, 
+            //     overlay.attrs.top_left_lat, 
+            //     overlay.attrs.bottom_right_lng, 
+            //     overlay.attrs.bottom_right_lat
+            // ];
+            // define the source extent (units = pixels) and targetExtent (EPSG:3857, units = meters)
+            // let extent = sourceLayer.getTileGrid().getExtent();
+            // let targetExtent = sourceLayer.getTileGrid().getExtent();
+
+            // // specify the point resolution in meters through custom function 
+            // // (default transforms the point from pixel to EPSG:4326, units = degrees)
+            // projection.setGetPointResolution(
+            //     (r) => r * Math.max(
+            //                 getWidth(targetExtent) / getWidth(extent),
+            //                 getHeight(targetExtent) / getHeight(extent)
+            //             )
+            // );
+
+            // // add coordinate transforms between the source-projection and target projection (same as view-projection)
+            // addCoordinateTransforms(
+            //     projection,
+            //     'EPSG:3857',
+            //     ([x, y]) => [
+            //         targetExtent[0] +
+            //         ((x - extent[0]) * getWidth(targetExtent)) / getWidth(extent),
+            //         targetExtent[1] +
+            //         ((y - extent[1]) * getHeight(targetExtent)) / getHeight(extent),
+            //     ],
+            //     ([x, y]) => [
+            //         extent[0] +
+            //         ((x - targetExtent[0]) * getWidth(extent)) / getWidth(targetExtent),
+            //         extent[1] +
+            //         ((y - targetExtent[1]) * getHeight(extent)) / getHeight(targetExtent),
+            //     ]
+            // );
+            
+            let tileLayer = new TileLayer({
+                source: sourceLayer,
+            });
+
+            return tileLayer;
+        },
+        updateCurrentImage(id, image) {
+            this.currentImage = image;
```

Comment:
@mzur,
when metadata is provided for the volume, what is the currently implemented way to access the metadata for each image (i.e. the point geo-coordinate)?

Reply:
The [image model](https://biigle.github.io/schema/tables/images.html) has the attributes `lat` and `lng`. Other metadata must be accessed via `$image->metadata`.